R/rextendr_document.R

```diff
@@ -9,13 +9,13 @@
 #' @inheritParams devtools::document
 #' @return No return value, called for side effects.
 #' @export
-document <- function(pkg = ".", quiet = getOption("usethis.quiet", FALSE), roclets = NULL) {
+document <- function(pkg = ".", quiet = FALSE, roclets = NULL) {
   try_save_all(quiet = quiet)
 
   withr::local_envvar(devtools::r_env_vars())
 
   register_extendr(path = pkg, quiet = quiet)
 
   rlang::check_installed("devtools")
-  devtools::document(pkg = pkg, roclets = roclets, quiet = FALSE)
+  devtools::document(pkg = pkg, roclets = roclets, quiet = quiet)
```

Comment:
@multimeric @Ilia-Kosenkov I inferred here that it would be desirable to silence `devtools::document()` when `quiet = TRUE`. Can either of you confirm that this is indeed desired? Otherwise it can be reverted.

Reply:
Makes sense to me.